app/streamSelector/browser.js

```diff
@@ -0,0 +1,87 @@
+const { desktopCapturer, ipcRenderer } = require("electron");
+window.addEventListener("DOMContentLoaded", () => {
+  ipcRenderer.once('get-screensizes-response', (event, screens) => {
```

Comment:
This function is a bit big. can you split it into smaller ones as it will make it easier to read/understand/maintain. Thanks

Reply:
Yes, I'd do some refactoring. Thanks!